tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -8,198 +8,225 @@
 using Nuke.Common.Tooling;
 using Nuke.Common.Tools.Git;
 using static Nuke.Common.Tools.DotNet.DotNetTasks;
+using Target = Nuke.Common.Target;
 
 partial class Build
 {
     AbsolutePath ExplorationTestsDirectory => RootDirectory / "exploration-tests";
 
+    [Parameter("Indicates use case of exploration test to run.")]
+    readonly ExplorationTestUseCase? ExplorationTestUseCase;
+
     [Parameter("Indicates name of exploration test to run. If not specified, will run all tests sequentially.")]
     readonly ExplorationTestName? ExplorationTestName;
 
-    [Parameter("Indicates whether exploration tests should skip repository cloning. Useful for local development. Default false.")]
-    readonly bool ExplorationTestSkipClone;
-
     [Parameter("Indicates whether exploration tests should run on latest repository commit. Useful if you want to update tested repositories to the latest tags. Default false.", 
                List = false)]
     readonly bool ExplorationTestCloneLatest;
 
-    Target RunExplorationTests_Debugger
+    Target SetUpExplorationTests
         => _ => _
-               .Description("Run exploration tests for debugger.")
+               .Description("Setup exploration tests.")
+               .Requires(() => ExplorationTestUseCase != null)
```

Comment:
Is `ExplorationTestUseCase  == ""` valid? if not you can use this:
```suggestion
               .Requires(() => ExplorationTestUseCase)
```